test/test_other.py

```diff
@@ -14269,6 +14269,44 @@ def test_windows_batch_file_dp0_expansion_bug(self):
     create_file('build_with_quotes.bat',  f'@"emcc" {test_file("hello_world.c")}')
     self.run_process(['build_with_quotes.bat'])
 
+  @only_windows('Check that directory permissions are properly retrieved on Windows')
+  @requires_node
+  def test_windows_nodefs_execution_permission(self):
+    if self.get_setting('WASMFS'):
+      self.set_setting('FORCE_FILESYSTEM')
+    delete_dir('new-dir')
+    src = r'''
+    #include <assert.h>
+    #include <emscripten.h>
+    #include <sys/stat.h>
+    #include <string.h>
+
+    int main(int argc, char * argv[]) {
+      EM_ASM(
+        FS.mkdir('/working');
+        FS.mount(NODEFS, { root: '{{WORKING_DIR}}' }, '/working');
```

Comment:
I think you can just do `{ root: '.' }` here?  Unless I'm missing something?

Reply:
Ah, true. I think I introduced this early on when I was running the test without using the test runner. Now using the test runner, I confirmed that using `{ root: '.' }` works as expected. I'll update it.